compiler/src/iree/compiler/Bindings/Native/Transforms/Passes.cpp

```diff
@@ -8,13 +8,18 @@
 
 #include <memory>
 
+#include "iree/compiler/Dialect/Util/IR/UtilOps.h"
+#include "iree/compiler/Utils/PassUtils.h"
 #include "mlir/Dialect/Func/IR/FuncOps.h"
 #include "mlir/Pass/PassOptions.h"
 #include "mlir/Pass/PassRegistry.h"
 #include "mlir/Transforms/Passes.h"
 
 namespace mlir::iree_compiler::IREE::ABI {
 
+using FunctionLikeNest =
+    MultiOpNest<func::FuncOp, IREE::Util::InitializerOp, IREE::Util::FuncOp>;
```

Comment:
Could this be written to use FunctionOpInterface instead of an explicit enumeration?

Reply:
unfortunately not yet - the pass manager can't nest on interfaces so if you add the pass it can't parallelize and will run at the module scope serially ;(